clients/spark/README.md

```diff
@@ -106,3 +106,8 @@ To do it, you should trigger the [GitHub Action](https://github.com/treeverse/la
    ```bash
    make publish-scala
    ```
+   
+### Debugging
```

Comment:
nit:
```suggestion
## Debugging
```
(this is _not_ a subsection of "publishing a new version", so it needs to be at level 2 not 3).